docs/manual/config/without-dm.rst

```diff
@@ -0,0 +1,64 @@
+====================
+Running from systemd
+====================
+
+This case will cover automatic login to Qtile after booting the system without
+using display manager directly from **systemd**. It logins in virtual console
+and init X by running through session.
+
+Automatic login to virtual console
+----------------------------------
+
+To get login into virtual console as an example edit `getty` service by running
+`systemctl edit getty@tty1` and add instructions:
+
+::
+    :caption: /etc/systemd/system/getty@tty1.service.d/override.conf
+
+    [Service]
+    ExecStart=
+    ExecStart=-/usr/bin/agetty --autologin username --noclear %I $TERM
+
+`username` should be changed to current user name.
+
+Check more for other `examples <https://wiki.archlinux.org/index.php/Getty#Automatic_login_to_virtual_console>`_.
+
+Qtile session
+-------------
+
+Starting Qtile directly will not preserve running applications on Qtile
+restarts by `qtile-cmd -o cmd -f restart` or other ways in Qtile itself. This
```

Comment:
I don't have this problem?

Reply:
Are you sure?

Reply:
Well, yes, unless I don't understand correctly "preserve running applications". Isn't it just related to #1615?

Reply:
No, Qtile can crash for many reasons, not just that one :)

Reply:
FOr me, restarting Qtile also restores my running applications correctly. Sometimes not in the same order/position, but restored anyway :slightly_smiling_face:

Reply:
@tych0 it's not about crashes here, but simple restarts

Reply:
You're right that I think the text is wrong, but the problem still exists: if Qtile crashes unexpectedly, without something to restart it, the X session will end.

Reply:
Which is to be expected, isn't it? Do you imply that we have a way to keep running applications even in the case of a qtile crash?